### DISCUSSION TITLE
CosmosOS and JetBrains Rider

### DISCUSSION THREAD

**Discussion by DZZc0rd** [CosmosOS/Cosmos#2806] [Q&A]
Hello, I want to make my os, but I don't know can I make os with CosmosOS with Rider?

**Comment by zarlo** [comment]
yes, you will need to disable the debugger but

**Reply by IMBA-Dev** [reply]
Could you write how to do this?

**Comment by 9xbt** [comment]
i usually create it in vs and then just open it up in rider and work on it there
for building just run dotnet build and them open a vmware vm with the iso

**Reply by zarlo** [reply]
if you install the dotnet templates it will show up in rider

**Reply by notmithun** [reply]
> if you install the dotnet templates it will show up in rider

Where do I get dotnet templates?

**Reply by zarlo** [reply]
`dotnet new --install ./source/templates/csharp/` from in the cosmos git folder

https://cosmosos.github.io/articles/Installation/DevKit.html#dotnet-project-templates

**Reply by notmithun** [reply]
> `dotnet new --install ./source/templates/csharp/` from in the cosmos git folder
> 
> https://cosmosos.github.io/articles/Installation/DevKit.html#dotnet-project-templates

Thank you
